pkg/services/team/teamimpl/store.go

```diff
@@ -454,6 +455,16 @@ func removeTeamMember(sess *db.Session, cmd *team.RemoveTeamMemberCommand) error
 	return err
 }
 
+// RemoveUsersMemberships removes all the team membership entries for the given user.
+// Only used when removing a user from a Grafana instance.
+func (ss *xormStore) RemoveUsersMemberships(ctx context.Context, userID int64) error {
+	return ss.db.WithTransactionalDbSession(ctx, func(sess *db.Session) error {
+		var rawSQL = "DELETE FROM team_member WHERE user_id = ?"
+		_, err := sess.Exec(rawSQL, userID)
+		return err
+	})
+}
```

Comment:
Note: this code has been moved over from [team guardian store](https://github.com/grafana/grafana/pull/68841/files#diff-eaaf4c694d714e9b77581e7868871dfd9e15a5eaa7cc807430880bf53e449e10L19-L25).